ui/app/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -149,7 +147,6 @@ export default class ConfirmPageContainer extends Component {
             subtitle={subtitle}
             subtitleComponent={subtitleComponent}
             hideSubtitle={hideSubtitle}
-            summaryComponent={summaryComponent}
             detailsComponent={detailsComponent}
             dataComponent={dataComponent}
             errorMessage={errorMessage}
```

Comment:
This file looks good as per https://github.com/MetaMask/metamask-extension/pull/10295/files#r566206507